lib/ZwaveClient.js

```diff
@@ -156,6 +156,9 @@ async function init (cfg, socket) {
     this.client.updateOptions(options)
   } else {
     this.client = CLIENT = new OpenZWave(options)
+    if (cfg.plugin) {
```

Comment:
What about also make a check the plugin path exists? `fs.existSync`?

Reply:
@chrisns What about this?

Reply:
agree, will push a fix later today

Reply:
put it in a try/catch instead of your recommendation since this allows normal manipulation of the paths that require uses (which might come in handy when we write tests) and felt like a better practice.

Reply:
It's ok for me :)